modules/migrations/gitea_downloader.go

```diff
@@ -101,12 +101,9 @@ func NewGiteaDownloader(ctx context.Context, baseURL, repoPath, username, passwo
 	// set small maxPerPage since we can only guess
 	// (default would be 50 but this can differ)
 	maxPerPage := 10
-	// new gitea instances can tell us what maximum they have
-	if giteaClient.CheckServerVersionConstraint(">=1.13.0") == nil {
-		apiConf, _, err := giteaClient.GetGlobalAPISettings()
-		if err != nil {
-			return nil, err
-		}
+	// gitea instances >=1.13 can tell us what maximum they have
+	apiConf, _, _ := giteaClient.GetGlobalAPISettings()
+	if apiConf != nil {
```

Comment:
But we still need to log the third return parameter error?

Reply:
I dont understand?

Reply:
@6543 I think lunny meant to log `err` parameter from `apiConf, _, err := giteaClient.GetGlobalAPISettings()`

Reply:
Ah, no we dont ...

Reply:
To ignore this error is the FIX :)

And since we ignore, we do not need a version check ... so remove the version chekc is a Refactor

Reply:
hmm... I disagree you should be logging this error and stating you are ignoring it at log.Info level (or at least log.Debug) in case this is fundamental to future problems.